code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -267,10 +270,13 @@ var/list/robot_verbs_default = list(
 	if(module)
 		return
 	var/list/modules = list("Standard", "Engineering", "Medical", "Miner", "Janitor", "Service")
-	if(!config.forbid_secborg)
-		modules += "Security"
-	if(!config.forbid_peaceborg)
-		modules += "Peacekeeper"
+	if(ert_upgrade)
+		modules = list("Engineering", "Medical", "Security")
+	else

```

Comment:
Why add this `else` instead of just putting the `if(ert_upgrade)` below these if statements?